<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>431</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Joda</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/dinosaur.jpeg</normaloff>../../../Downloads/dinosaur.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(43, 43, 43)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 55, 252);
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Escanear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Apelldo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>94</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSur">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>94</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Apellido</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblId">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Hora Entrada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>210</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 55, 252)</string>
    </property>
    <property name="text">
     <string>Invitado en Puerta</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 55, 252)</string>
    </property>
    <property name="text">
     <string>Esperados</string>
    </property>
   </widget>
   <widget class="QListView" name="listView2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 55, 252)</string>
    </property>
    <property name="text">
     <string>Adentro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 55, 252);
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTim">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>Sexo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSex">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Sexo</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(43, 43, 43)</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(43, 43, 43)</string>
   </property>
   <widget class="QMenu" name="menuFunciones">
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(43, 43, 43)</string>
    </property>
    <property name="title">
     <string>Funciones</string>
    </property>
    <addaction name="escan"/>
    <addaction name="agre"/>
    <addaction name="resu"/>
    <addaction name="ingreList"/>
    <addaction name="delet"/>
   </widget>
   <addaction name="menuFunciones"/>
  </widget>
  <action name="escan">
   <property name="text">
    <string>Escanear</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="agre">
   <property name="text">
    <string>Agregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="resu">
   <property name="text">
    <string>Resumen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="ingreList">
   <property name="text">
    <string>Ingresar Lista</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="delet">
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>listView2</tabstop>
  <tabstop>listView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>abrirAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>escanear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>abrirAdd()</slot>
  <slot>escanear()</slot>
 </slots>
</ui>
